Source/WebGPU/WebGPU/RenderPassEncoder.mm

```diff
@@ -386,7 +386,9 @@ static void setViewportMinMaxDepthIntoBuffer(auto& fragmentDynamicOffsets, float
         auto it = m_vertexBuffers.find(bufferIndex);
         RELEASE_ASSERT(it != m_vertexBuffers.end());
         auto bufferSize = it->value.size;
-        if ((strideCount.value() - 1) * bufferData.stride + bufferData.lastStride > bufferSize) {
+        auto strideSize = checkedProduct<uint64_t>((strideCount.value() - 1), bufferData.stride);
+        auto requiredSize = checkedSum<uint64_t>(strideSize.value());
```

Comment:
I think the above line should read:
```
auto requiredSize = checkedSum<uint64_t>(strideSize.value(),  bufferData.lastStride);
```
otherwise `bufferData.lastStride` is removed from the calculation, right?

Reply:
This has been corrected.